pkg/volume/plugins.go

```diff
@@ -121,6 +122,9 @@ type VolumeHost interface {
 	// the provided spec.  See comments on NewWrapperBuilder for more
 	// context.
 	NewWrapperCleaner(spec *Spec, podUID types.UID, mounter mount.Interface) (Cleaner, error)
+
+	//Get cloud provider from kubelet
+	GetCloudProvider() cloudprovider.Interface
```

Comment:
@saad-ali openstack couldn't instantiate itself, so I submitted a different patch to make kubelet cloudprovider aware. How ever without adding it to the volume host, the only other way was to pass it into the builder, which also didn't seem correct.